Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp

```diff
@@ -302,6 +302,24 @@ bool AsyncScrollingCoordinator::requestStopKeyboardScrollAnimation(ScrollableAre
     return true;
 }
 
+void AsyncScrollingCoordinator::mouseDidEnterScrollbar(ScrollableArea& scrollableArea, bool didEnterScrollbar)
```

Comment:
/* Unofficial Review */
I think either removing didEnterScrollbar or changing the method name is needed. The method name says the mouse entered scrollbar though, why do we need a boolean for it?

Reply:
I think this name is reasonable as the bool represents if the mouse is entering or exiting the scrollbar.